src/components/message-input/index.test.tsx

```diff
@@ -155,29 +155,30 @@ describe('MessageInput', () => {
   });
 
   it('searches for matching users via userMentionSearch function', async function () {
-    const getUsersForMentions = async (_searchString) => Promise.resolve([{ id: '1', display: 'dale' }]);
+    const getUsersForMentions = async (_searchString) =>
+      Promise.resolve([{ id: '1', display: 'dale', profileImage: 'http://example.com' }]);
     const wrapper = subject({ getUsersForMentions });
 
     const searchResults = await userSearch(wrapper, 'da');
 
-    expect(searchResults).toEqual([{ display: 'dale', id: '1' }]);
+    expect(searchResults).toEqual([{ display: 'dale', id: '1', profileImage: 'http://example.com' }]);
   });
 
   it('sorts by search string index', async function () {
     const getUsersForMentions = async (_searchString) =>
       Promise.resolve([
-        { id: 'd-2', display: '2-dale' },
-        { id: 'd-3', display: '3--dale' },
-        { id: 'd-1', display: 'dale' },
+        { id: 'd-2', display: '2-dale', profileImage: 'http://example-2.com' },
```

Comment:
FYI, we should use only `example.com` here (not `example-1`, etc). `example.com` is the only reserved domain that nobody can register and we wouldn't want to be accidentally downloading something from somewhere else. So, `http://example.com/1` , etc, would be more appropriate.

Note: I'm sure I/we have made this mistake all over the place so if you do run into others it'd be nice to fix up.